SalesforceSDK/Core/Auth/OAuth2.cs

```diff
@@ -323,30 +298,53 @@ public static string ComputeFrontDoorUrl(string instanceUrl, string displayType,
         /// <returns>Boolean based on if the refresh auth token succeeded or not</returns>
         public static async Task<Account> RefreshAuthToken(Account account)
```

Comment:
Yep - now's a good time to line up the API names. We don't want to change them later.